apps/core/scenes/passengerForm/BaggageBundles/BaggageBundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, RadioButton } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BaggageTypeDetail from './BaggageTypeDetail';
+
+type Props = {|
+  +price: ?string,
+  +bags: any,
+  +bagIndex: number,
+  +onToggleCheck: any,
+  +type: string,
+  +currentlyChosen: number,
```

Comment:
Right now, you are using static data, but maybe it could be nullable in the future, what do you think? I'm not sure

Reply:
Hmm what do you think if its going to be changed once becoming fragment, as for now it not nullable yet?

Reply:
OK, no problem